daphne/src/roles_test.rs

```diff
@@ -614,42 +581,24 @@ async fn get_reports_empty_response() {
         .await
         .expect("upload failed unexpectedly");
 
-    // Get report.
-    let now = report.nonce.time - task_config.min_batch_duration - 1;
+    // We should get one report.
```

Comment:
Why? The The name of the test is `get_reports_empty_response()`. Also, it looks like the output is not examiined. Let's take a step back and ask ourselves what this test case is for.

Reply:
Updated the test case.